featured_lists.json

```diff
@@ -38,5 +38,6 @@
   "NGVO/NGVO",
   "GhoulifiedReality/GhoulifiedReality",
   "Listonomicon/Listonomicon",
-  "Palette/Alpyne"
+  "Palette/Alpyne",
+  "Baldur-s-Gate-3-Enhanced-Edition/reyqune_Baldur_s_Gate_3_Enhanced_Edition"
```

Comment:
this is the wrong repository name, see here:
https://github.com/wabbajack-tools/mod-lists/blob/82c584561b1d4d2ed5af5780f2743255fff0bd1d/repositories.json#L75C3-L75C11